src/client/java/minicraft/saveload/Load.java

```diff
@@ -492,6 +451,48 @@ private void loadPrefsOld(String filename) {
 				MultiplayerDisplay.savedUsername = data.remove(0);
 			}
 
+			// Get legacy language and convert it into the current format.
+			if (prefVer.compareTo(new Version("2.0.4-dev3")) >= 0) {
+				// Get language and convert into locale.
+				String lang;
+				switch (data.remove(0)) {
+					case "english":
+						lang = "en-us";
+						break;
+					case "french":
+						lang = "fr-fr";
+						break;
+					case "hungarian":
+						lang = "hu-hu";
+						break;
+					case "indonesia":
+						lang = "id-id";
+						break;
+					case "italiano":
+						lang = "it-it";
+						break;
+					case "norwegian":
+						lang = "nb-no";
+						break;
+					case "portugues":
+						lang = "pt-pt";
+						break;
+					case "spanish":
+						lang = "es-es";
+						break;
+					case "turkish":
+						lang = "tr-tr";
+						break;
+					default:
+						lang = null;
+						break;
+				}
+
+				if (lang != null) {
+					Localization.changeLanguage(lang);
+				}
+			}
```

Comment:
You're aware that this block will never run this code, right?

Reply:
This code might be run.

Reply:
It checks if the preferences version is less than 2.0.3-dev1 and more or equals to 2.0.4-dev3. That will never run.

Reply:
There is an `else` with the outer `if` block, so it is greater or equals than 2.0.3-dev1.

Reply:
Good!